samples/AspNetCore/GrpcServiceSample/Services/BankingService.cs

```diff
@@ -0,0 +1,168 @@
+                        await Deposit(transaction, context) :
+                        await Withdraw(transaction, context);
+                    respone.Data = TypeConverters.ToAny<Account>(account, this.jsonOptions);
+                    break;
+                default:
+                    break;
+            }
+            return respone;
+        }
+
+        /// <summary>
+        /// implement ListTopicSubscriptions to register deposit and withdraw subscriber
+        /// </summary>
+        /// <param name="request"></param>
+        /// <param name="context"></param>
+        /// <returns></returns>
+        public override Task<ListTopicSubscriptionsResponse> ListTopicSubscriptions(Empty request, ServerCallContext context)
+        {
+            var result = new ListTopicSubscriptionsResponse();
+            result.Subscriptions.Add(new TopicSubscription
+            {
+                PubsubName = "pubsub",
+                Topic = "deposit"
+            });
+            result.Subscriptions.Add(new TopicSubscription
+            {
+                PubsubName = "pubsub",
+                Topic = "withdraw"
+            });
+            return Task.FromResult(result);
+        }
+
+        /// <summary>
+        /// implement OnTopicEvent to handle deposit and withdraw event
+        /// </summary>
+        /// <param name="request"></param>
+        /// <param name="context"></param>
+        /// <returns></returns>
+        public override async Task<TopicEventResponse> OnTopicEvent(TopicEventRequest request, ServerCallContext context)
+        {
+            if (request.PubsubName == "pubsub")
+            {
+                var transaction = JsonSerializer.Deserialize<Transaction>(request.Data.ToStringUtf8(), this.jsonOptions);
+                if (request.Topic == "deposit")
+                    await Deposit(transaction, context);
+                else
+                    await Withdraw(transaction, context);
+            }
+           
+            return await Task.FromResult(default(TopicEventResponse));
+        }
+
+        /// <summary>
+        /// GetAccount
+        /// </summary>
+        /// <param name="input"></param>
+        /// <param name="context"></param>
+        /// <returns></returns>
+        public async Task<Account> GetAccount(GetAccountInput input, ServerCallContext context)
+        {
+            var state = await _daprClient.GetStateEntryAsync<Account>(StoreName, input.Id);
+
+            if (state.Value == null)
+            {
+                return null;
+            }
+
+            return state.Value;
+        }
+
+        /// <summary>
+        /// Deposit
+        /// </summary>
+        /// <param name="transaction"></param>
+        /// <param name="context"></param>
+        /// <returns></returns>
+        public async Task<Account> Deposit(Transaction transaction, ServerCallContext context)
+        {
+            Console.WriteLine("Enter deposit");
```

Comment:
Does this have a purpose? if so, it might be better to use the logger rather than directly writing to the console

Reply:
I use logger to replace Console.WriteLine. Actually, I borrow this line from ControllerSample, and I also fixed it in ControllerSample.